airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/metadata.py

```diff
@@ -149,8 +149,33 @@ def _run(self, connector: Connector) -> CheckResult:
         )
 
 
+class CheckConnectorMaxSecondsBetweenMessagesValue(MetadataCheck):
+    name = "Certified connector must have a value filled out for maxSecondsBetweenMessages in metadata"
+    description = f"Certified connectors must have a value filled out for maxSecondsBetweenMessages in metadata. It must be set in the 'data' field in {consts.METADATA_FILE_NAME}"
+    applies_to_connector_languages = [ConnectorLanguage.PYTHON, ConnectorLanguage.LOW_CODE]
+
+    @staticmethod
+    def check_connector_certified(connector: Connector) -> bool:
```

Comment:
I think this check should run on all connectors.

Reply:
All certified should be covered, what is the purpose of running this on the community sources?

Reply:
We want to enforce best practices on new community PRs too.

Reply:
I don't have a strong opinion. Say we run this on all community connectors — will their CI be red on all pull requests moving forward unless we add the field? We'd end up just adding the field with some dummy value, because knowing actual value requires tedious work of _actually reading the API docs_. 

I'd say it should be a requirement to be published on Cloud, so perhaps certified is alright?